python/ray/data/preprocessors/chain.py

```diff
@@ -94,3 +95,10 @@ def _transform_batch(self, df: "DataBatchType") -> "DataBatchType":
     def __repr__(self):
         arguments = ", ".join(repr(preprocessor) for preprocessor in self.preprocessors)
         return f"{self.__class__.__name__}({arguments})"
+
+    def _determine_transform_to_use(self, data_format: "BlockFormat") -> "BatchFormat":
```

Comment:
Do the type hints need to be in quotes here? The base preprocessor class doesn't have this.

Reply:
would prefer to keep this signature exactly the same as the base class.

Reply:
This is for postponed evaluation due to imports being in TYPE_CHECKING (would throw an exception otherwise)